.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
+  no-cond-assign:
+  - warn
+  - except-parens
+  no-confusing-arrow: off
+  no-console:
+  - warn
+  - allow:
+    - warn
+    - error
+  no-mixed-operators: off
+  no-nested-ternary: off
+  no-plusplus: warn
+  no-restricted-syntax: off
+  no-shadow: off
+  no-undef: warn
+  no-unused-vars: warn
```

Comment:
I turned [this rule](https://eslint.org/docs/rules/no-unused-vars) from an error into a warning because I think people prefer to write their React components with `props` even if they haven't figured out what to do with `props` yet. No preference here.